python/lsst/afw/image/exposure.py

```diff
@@ -0,0 +1,14 @@
+from __future__ import absolute_import
+
+from ._exposure import *
+
+def __reduce__(self):
+    from lsst.afw.fits import reduceToFits
+    return reduceToFits(self)
+
+ExposureI.__reduce__ = __reduce__
+ExposureF.__reduce__ = __reduce__
+ExposureD.__reduce__ = __reduce__
+ExposureU.__reduce__ = __reduce__
+#ExposureL.__reduce__ = __reduce__
```

Comment:
I don't think the two are related. RFC-227 is talking about how afw::table allows users to alias the C++ types (I, L, F, etc) using python types (int, long, float, etc). But in python 3 all ints are longs, so the C++ types can no longer be aliased using int and long, but instead np.int32 and np.int64. I don't think that this is related to that.